### PR TITLE
(maint) Update readme to reflect state of develpoment

### PR DESCRIPTION
**Pull request by jtappa**
We had a PR sitting in this repo for 6+ months with no reviews, so I thought this might be helpful to point out in the README.

### CODE REVIEW

**Comment by kris-bosland** [comment]
Hmm @jtappa, I feel like the first sentence is a little awkward so I am trying to figure out how to help improve it, but it also just occurred to me: maybe we should just archive this repo like they have been doing with the push for CODEOWNERS?

**Comment by jtappa** [comment]
Because we still have Puppet 4.x and 5.x branches, that's not possible. Not to mention the purpose of the repo being here is so that it can be a module available on the forge. The best solution would be if we actually had someone to maintain it until it could officially be archived.

File: README.md [hunk 1]
```diff
@@ -19,8 +19,6 @@ All `nagios_*` types default to having a `target` under `/etc/nagios/`, but shou
 <a id="development"></a>
 ## Development
 
-Puppet Labs modules on the Puppet Forge are open projects, and community contributions are essential for keeping them great. We can't access the huge number of platforms and myriad of hardware, software, and deployment configurations that Puppet is intended to serve.
+When the `nagios_*` types were removed from Puppet in Puppet Platform 6 and extracted to this module, they were effectively deprecated and are now no longer under active develepment.
 
-We want to keep it as easy as possible to contribute changes so that our modules work in your environment. There are a few guidelines that we need contributors to follow so that we can have a chance of keeping on top of things.
-
-For more information, see our [module contribution guide.](https://docs.puppetlabs.com/forge/contributing.html)
+While it's possible to follow our usual contribution guidelines for modules, we do not currently have a team dedicated to reviewing and merging pull requests for this repository and your PR could sit indefinitely waiting for review. As an alternative you can fork this repository, and make changes to your own forked version.
```
Comment (kris-bosland) [inline README.md]: How about 'While this repo is open for PRs following our usual contribution guidelines for modules,'?  It sounds a bit clearer to me what is meant initially, but maybe it also sounds more open than what we want.

Comment (jtappa) [reply README.md]: Yeah, I also dug around the settings and I couldn't quite figure it out but I almost wonder if there's a way we can just turn off PRs for this without archiving? or PRs that don't come from the puppetlabs org? @shrug do you know how to do something like that with your fancy admin access?

**Comment by jtappa** [comment]
@kris-bosland I updated this PR based on Josh's comment in slack:

>So let's archive the repos with instructions for how to get in contact on slack. If someone actually wants to improve them (AFAIK the nagios was the one and only PR in the last 9months or so?) , then we can revisit unarchiving and working with community to figure out a process.

So it seems like the path forward is to update the READMEs on the deprecated _core modules, and then archive them, which I'm happy to take care of the rest once we agree on a README message.

**Review by kris-bosland — APPROVED** [review]

**Pull request merged**